website/content/docs/connect/gateways/api-gateway/tech-specs.mdx

```diff
@@ -0,0 +1,157 @@
+
+You can deploy API gateways to Kubernetes clusters managed by Red Hat OpenShift, which is a security-conscious, opinionated wrapper for Kubernetes. To enable OpenShift support, add the following parameters to your Consul values file and apply the configuration:
+
+```yaml
+  openshift:
+    enabled: true
+ ```
+
+Refer to the following topics for additional information:
+
+- [Install Consul on OpenShift clusters with Helm](/consul/docs/k8s/installation/install#install-consul-on-openshift-clusters)
+- [Install Consul on OpenShift clusters with the `consul-k8s` CLI](/consul/docs/k8s/installation/install-cli#install-consul-on-openshift-clusters)
+
+### Security context constraints
+
+OpenShift requires a security context constraint (SCC) configuration, which restricts pods to specific groups. You can create a custom SCC or use one of the default constraints. Refer to the [OpenShift documentation](https://docs.openshift.com/container-platform/4.13/authentication/managing-security-context-constraints.html) for additional information. 
+
+By default, the SCC is set to `restricted-v2` for the `managedGatewayClass` that Consul automatically creates. The `restricted-v2` SCC is one of OpenShifts default SCCs, but you can specify a different SCC in the `openshiftSccName` parameter:
+
+```yaml
+connectInject:
+  apiGateway:
+    managedGatewayClass:
+      openshiftSccName: "restricted-v2"
+```
+
+### Privileged container ports
+ 
+Containers cannot use privileged ports when OpenShift is enabled. Privileged ports are 1 through 1024, and serving applications from that range is a security risk.
+
+To allow gateway listeners to use privileged port numbers, specify an integer value in the `mapPrivilegedContainerPorts` field of your Consul values configuration. Consul adds the value to listener port numbers that are set to a number in the privileged container range. Consul maps the configured port number to the total port number so that traffic sent to the configured port number is correctly forwarded to the service.
+
+For example, if a gateway listener is configured to port `80` and the `mapPrivilegedContainerPorts` field is configured to `2000`, then the actual port number on the underlying container is `2080`. 
+
+You can set the `mapPrivilegedContainerPorts` parameter in the following map in your Consul values file:
+
+```yaml
+connectInject:
+  apiGateway:
+    managedGatewayClass:
+      mapPrivilegedContainerPorts: <value>
+```
+
+## Supported versions of the Kubernetes gateway API specification
+
+Refer to the [release notes](/consul/docs/release-notes) for your version of Consul.
+
+## Supported Kubernetes gateway specification features
+
+Consul API gateways for Kuberentes support the following subset of the Kubernetes [v1alpha2 Gateway API specification](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/). For a complete list of features, including the list of gateway and route statuses and an explanation on how they
```

Comment:
```suggestion
Consul API gateways for Kuberentes support a subset of the Kubernetes Gateway API specification. For a complete list of features, including the list of gateway and route statuses and an explanation on how they
```
This is the simplest way to handle it. We can open a new PR and provide more detail before the next GA